src/components/footer/DonateButton.js

```diff
@@ -19,11 +17,11 @@ function DonateButton({ t }) {
         src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
         border="0"
         name="submit"
-        alt={t('footer-donate-paypal')}
+        alt={'footer-donate-paypal'}
```

Comment:
We are not using this component yet, therefore I didn't set the i18n yet. When putting it back, we should set this value.